relayer-engine/src/storage/storage.ts

```diff
@@ -45,23 +41,15 @@ export async function createStorage(
   logger?: Logger,
 ): Promise<Storage> {
   switch (config.storeType) {
```

Comment:
the switch statement is not necessary, we could add it again once we support more than one store type